.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
     - REPO=docksal/apache
 
   matrix:
-    - VERSION=2.4 FROM=httpd:2.4.35-alpine TAGS=2.4,latest
```

Comment:
It looks like `latest` tag will not get pushed. The `is_latest()` check in the `docker-push.sh` script will never pass.

Reply:
in travis.yml added LATEST_VERSION variable